exchanges/bybit/bybit_test.go

```diff
@@ -3307,3 +3307,47 @@ func TestExtractCurrencyPair(t *testing.T) {
 		t.Fatalf("expecting %v, got %v", dogeUSDT, pair)
 	}
 }
+
+func TestStringToOrderStatus(t *testing.T) {
+	t.Parallel()
+	input := []struct {
+		OrderStatus string
+		Expectation order.Status
+	}{
+		{
+			OrderStatus: "",
+			Expectation: order.UnknownStatus,
+		},
+		{
+			OrderStatus: "UNKNOWN",
+			Expectation: order.UnknownStatus,
+		},
+		{
+			OrderStatus: "Cancelled",
+			Expectation: order.Cancelled,
+		},
+		{
+			OrderStatus: "ACTIVE",
+			Expectation: order.Active,
+		},
+		{
+			OrderStatus: "NEW",
+			Expectation: order.New,
+		},
+		{
+			OrderStatus: "FILLED",
+			Expectation: order.Filled,
+		},
+		{
+			OrderStatus: "UNTRIGGERED",
+			Expectation: order.Pending,
+		},
+	}
+	var oStatus order.Status
+	for x := range input {
+		oStatus = StringToOrderStatus(input[x].OrderStatus)
```

Comment:
For some reason now when I called GetOrderHistory Status is now being set as `Status:PARTIALLY_CANCELLED` when it should be set as cancelled now.

Reply:
Isn't the status message `PartiallyFilledCanceled`? If so, Not I created a new order.Status instance **PartiallyFilledCancelled**